argoverse/evaluation/stereo/disparity_interpolation/disparity_interpolation/source/disparity_interpolation.pyx

```diff
@@ -0,0 +1,40 @@
+# distutils: language = c++
+# distutils: sources = nn_interpolation.cpp
+# cython: language_level=3
+
+import cython
+
+import numpy as np
+cimport numpy as np
+
+
```

Comment:
How does this interpolation speed compare to what sklearn or scipy provides? Maybe you could add in the comment what the expected run time is for a typical stereo image, e.g. 2056 × 2464?

Would this be faster if we wrapped it in Pybind in pure C++ instead of Cython?

Would it possibly make sense to release this as a separate repo that we could put on pybind, so that nothing has to be compiled?

Reply:
Using the Scipy's interpolate.NearestNDInterpolator function it takes about 5 seconds. Using this Cython function it takes about 50 ms. It's 100x faster. 

Maybe pybind11 would be slightly faster. But I haven't yet tested.
I liked the idea of putting it on pypi, so we don't have to compile anything. So I removed the disparity_interpolation module and placed it on pypi. Now we simply need to `pip install disparity_interpolation`.